src/main/java/cam72cam/immersiverailroading/registry/EntityRollingStockDefinition.java

```diff
@@ -470,6 +471,9 @@ public void loadData(DataBlock data) throws Exception {
         DataBlock properties = data.getBlock("properties");
         weight = properties.getValue("weight_kg").asInteger() * internal_inv_scale;
         valveGear = ValveGearConfig.get(properties, "valve_gear");
+        if(valveGear !=  null){
+            isLeftFirst = properties.getValue("left_first").asBoolean(true);
```

Comment:
Instead of using .asBoolean(true), lets uncomment the value in default/base.caml.  I've tried to move to that system so the documentation and code don't get out of sync.

Reply:
> Instead of using .asBoolean(true), lets uncomment the value in default/base.caml. I've tried to move to that system so the documentation and code don't get out of sync.

Fixed it

Reply:
> Instead of using .asBoolean(true), lets uncomment the value in default/base.caml. I've tried to move to that system so the documentation and code don't get out of sync.

But sir won't this lead to some capability issues?